src/oidc/register.ts

```diff
@@ -35,16 +35,16 @@ export type OidcRegistrationClientMetadata = {
 };
 
 interface OidcRegistrationRequestBody {
```

Comment:
It can be great to have a link to the MSC doc. To know which spec/msc this type refers to